packages/block-library/src/heading/edit.native.js

```diff
@@ -57,12 +50,10 @@ class HeadingEdit extends Component {
 					tagName={ tagName }
 					value={ content }
 					isSelected={ this.props.isSelected }
+					style={ { ...style, minHeight } }
```

Comment:
Should not it be declared the other way round? `style={ { minHeight, ...style } }`

Otherwise if I set a style to a particular Heading block that has a minHeight different from the default value `const minHeight = styles.blockText.minHeight;` it will be override by this.

Reply:
You are correct let me correct this.